sympy/stats/tests/test_continuous_rv.py

```diff
@@ -149,6 +149,16 @@ def test_cdf():
     f = cdf(Z)
     assert f(z) == Piecewise((1 - exp(-z), z >= 0), (0, True))
 
+    X = Normal('x', 1, 0)
+    d = cdf(X)
+    assert d(0.9) == 0
+    assert d(1) == 1
+    # The following 2 lines should produce the same result,
+    # However, P(X <= 1) integrates the pdf instead of using the cdf
+    # which it should not for computational and efficiency purposes.
```

Comment:
I think integrating is fine, because there's no other choice for conditions of more than one variable.

The bug seems to be in `Integral` itself.

Reply:
Oh. I thought you were referring to a possible improvement where `P(RandomSymbol <= Expr)` should use the cdf. (I would have assumed it would increase performance since it won't need to integrate at all.) 

I'll remove the comments this time tomorrow.

Reply:
I suppose using the cdf would be faster, but it's not the cause of the issue.